lib/nodes/Switch.ts

```diff
@@ -0,0 +1,59 @@
+import {BlueshellState, ResultCode, isResultCode, rc, BaseNode, Conditional} from '../models';
+import {Parent} from './Parent';
+import {Constant} from './Constant';
+
+export interface SwitchEntry<S extends BlueshellState, E> {
```

Comment:
Maybe put a comment on this interface that conditional() should not perform any mutations?

Reply:
We have conditionals in a lot of places. If I were to do this, I would prefer to put it on the interface definition for it. What do you think about that?

Reply:
Oh, yes, even better!

Reply:
comments added in `lib/models/Conditional.ts` and `lib/models/ConditionalWithResult.ts`